### PR TITLE
ofrule: avoid zombies from ovs-ofctl commands

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
rerun tests

File: topology/probes/ovs_of.go [hunk 1]
```diff
@@ -265,6 +265,7 @@ var executor Execute = RealExecute{}
 func (r RealExecute) ExecCommand(com string, args ...string) ([]byte, error) {
 	/* #nosec */
 	command := exec.Command(com, args...)
+	defer command.Wait()
```
Comment (nplanel) [inline topology/probes/ovs_of.go]: CombinedOutput() call Run(), Run() call Wait()
https://golang.org/src/os/exec/exec.go?s=8356:8381#L275

Comment (safchain) [reply topology/probes/ovs_of.go]: yep good catch, will remove it

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun tests

**Pull request merged**